homeassistant/components/ai_speaker/sensor.py

```diff
@@ -27,23 +25,26 @@ async def async_unload_entry(hass, entry):
 class AisSensor(Entity):
     """AiSpeakerSensor representation."""
 
-    def __init__(self, hass, config_entry_data):
+    def __init__(self, hass, ais_gate_instance):
         """Sensor initialization."""
-        self._ais_info = config_entry_data.get("ais_info")
-        self._ais_id = self._ais_info.get("ais_id")
-        self._ais_url = self._ais_info.get("ais_url")
-        self._web_session = aiohttp_client.async_get_clientsession(hass)
-        self._ais_gate = AisWebService(hass.loop, self._web_session, self._ais_url)
+        self._ais_gate = ais_gate_instance
+        self._ais_info = None
```

Comment:
We don't need the general `_ais_info` attribute if we just use the specific ones below.

Reply:
true - thx :)
done in  e0890e4